code/modules/crafting/guncrafting.dm

```diff
@@ -81,3 +95,34 @@
 			to_chat(user, "<span class='warning'>You need at least five feet of wrapping paper to secure the stock.</span>")
 			return
 
+//Flare Rifle//
+
+/obj/item/weaponcrafting/flaregunconstruction
```

Comment:
Just on the side, but I thought we got rid of all the step construction things in favor for table crafting. Were firearm weapons excluded from this?

Reply:
Somehow they weren't gotten rid of, so I figured I would also add this including the hand crafting with the table crafting.